students/Z_shen/lesson04/mailroom_2.py

```diff
@@ -0,0 +1,143 @@
+donor = [
+    {'Name': 'William Gates', 'Donation': 653772.32, 'number_don': 2},
+    {'Name': 'Jeff Bezos', 'Donation': 163960000.10, 'number_don': 3},
+    {'Name': 'Paul Allen', 'Donation': 877.33, 'number_don': 1},
+    {'Name': 'Mark Zuckerberg', 'Donation': 708.42, 'number_don': 3},
+]
```

Comment:
expected to use a dictionary for your main data structure -- please convert your list to dictionary.

a good data structure would be:
```
donors = {"Bob Smith": [3,3, 33]}
```
or if you want to have it similar to yours:
```
donors = {"Bob Smith": {"donations": [33, 33], "number_of_donations": 3}}
```